presto-hive/src/test/java/com/facebook/presto/hive/TestHiveRecoverableGroupedExecution.java

```diff
@@ -64,7 +64,7 @@
 import static org.testng.Assert.assertEquals;
 import static org.testng.Assert.assertTrue;
 
-@Test(singleThreaded = true)
+@Test(singleThreaded = true, enabled = false)
```

Comment:
you need to disable each method individually because the method level @Test annotations will override the class level one.

Reply:
Oh actually what version of testng do we use.  Apparently there's now an @Ignore annotation https://github.com/cbeust/testng/issues/151#issuecomment-385381333

Reply:
@rschlussel good catch

Reply:
wow good catch!

Reply:
We are currently using 6.10. I'll try and see if we could bring it to 6.14.3.

Reply:
I checked the log and it seems that `@Ignore` works. I'll wait and see if the other tests would pass after the upgrade.

Reply:
Well now I see five test suites failed so I'm not sure if we are really ready for 6.14.3. I'll just disable them on method level then.